src/components/signup/Signin.jsx

```diff
@@ -1,3 +1,92 @@
-export function Signin() {
-  return <></>;
+import { useState } from "react";
+import "./signup.css";
+export function Signin({ setIsSignUp }) {
+  const [showPassword, setShowPassword] = useState(false);
+  const [loginForm, setLoginForm] = useState({
+    email: "",
+    password: "",
+  });
+
+  const changeHandler = event => {
+    event.preventDefault();
+    setLoginForm(() => ({
+      ...loginForm,
+      [event.target.name]: event.target.value,
+    }));
+  };
+
+  const toggleShowPassword = e => {
+    e.preventDefault();
+    setShowPassword(() => (showPassword ? false : true));
```

Comment:
```suggestion
    setShowPassword((showPassword) => !showPassword);
```

would be a better approach here.

Reply:
Oh yes, how could I forget that! Made the changes